pkg/printers/json.go

```diff
@@ -8,22 +8,30 @@ import (
 	"github.com/golangci/golangci-lint/pkg/result"
 )
 
-type JSON struct{}
+type JSON struct {
+	disableCongrats bool
+}
 
-func NewJSON() *JSON {
-	return &JSON{}
+func NewJSON(disableCongrats bool) *JSON {
+	return &JSON{
+		disableCongrats: disableCongrats,
+	}
 }
 
 type JSONResult struct {
 	Issues []result.Issue
 }
 
-func (JSON) Print(ctx context.Context, issues <-chan result.Issue) (bool, error) {
+func (j JSON) Print(ctx context.Context, issues <-chan result.Issue) (bool, error) {
 	allIssues := []result.Issue{}
 	for i := range issues {
 		allIssues = append(allIssues, i)
 	}
 
+	if len(allIssues) == 0 && j.disableCongrats {
+		return true, nil
```

Comment:
I know it's ugly and we need to refactor it. Here need to return `false`: it's a flag that issues were found

Reply:
Ah ok I missed the logic a bit here